### ISSUE TITLE
[facetoface] Discuss Accessibility Standards Process

### ISSUE THREAD

**Issue by dylan-fox** [immersive-web/webxr#1320]
WebXR is currently missing many core accessibility features, such as the ability to store and communicate metadata about discrete objects within an XR space. Some of these can be addressed in the WebXR APIs, but some may take collaboration with other W3C groups such as the XAUR. As a case study, we can consider the focus of the NSF Accelerator project worked on by XR Access, Benetech, and Lighthouse: creating guidelines about translating nonverbal communication in XR into sound and haptic signals to improve accessibility for blind and low-vision users.

XR Access is willing to facilitate communications about standards to platform owners, other W3C groups, and standards-gathering organizations like the Metaverse Standards Forum.
